src/main/java/com/synopsys/arc/jenkinsci/plugins/extratoolinstallers/installers/AnyOfInstaller.java

```diff
@@ -0,0 +1,307 @@
+package com.synopsys.arc.jenkinsci.plugins.extratoolinstallers.installers;

```

Comment:
If you prefer, you can start creating new code in another group id, e.g. `io.jenkins.plugins.extratoolinstallers.installers`

Reply:
Good idea.  Done.